client/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "react-webpack-rails-tutorial",
+  "version": "1.1.0",
+  "description": "Built using the react_on_rails generator.",
+  "main": "server.js",
+  "engines": {
+    "node": "5.0.0",
+    "npm": "3.5.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial.git"
+  },
+  "keywords": [
+    "react",
+    "tutorial",
+    "comment",
+    "example"
+  ],
+  "author": "justin808",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial/issues"
+  },
+  "homepage": "https://github.com/shakacode/react-webpack-rails-tutorial",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js",
+    "build:client": "NODE_ENV=production webpack --config webpack.client.rails.config.js",
+    "build:server": "NODE_ENV=production webpack --config webpack.server.rails.config.js",
```

Comment:
The tasks without `:dev` are used during asset precompilation to create the generated bundles for deployment, such as to Heroku.